gui/src/2D/controls/control.ts

```diff
@@ -25,6 +25,11 @@ export class Control {
      */
     public static AllowAlphaInheritance = false;
 
+    /** Gets the number of layout calls made the last time the ADT has been rendered */
+    public static NumLayoutCalls = 0;
```

Comment:
Should not be Static but per ADT

Reply:
Going to correct that: I think I will need to create a new PR?

Reply:
yes my dear :)